wix/windows-toolchain.wxs

```diff
@@ -395,6 +395,7 @@
     <DirectoryRef Id="USR_INCLUDE__INTERNALSWIFTSYNTAXPARSER">
       <?if $(var.HAVE__INTERNAL_SWIFT_SYNTAX_PARSER) = true ?>
         <Component Id="_INTERNALSWIFTSYNTAXPARSER_HEADERS" Guid="751f758a-ee88-4be5-9285-97bbf3aa374a">
+          <File Id="SWIFTSYNTAXCDATATYPES_H" Source="$(var.TOOLCHAIN_ROOT)\usr\lib\swift\_InternalSwiftSyntaxParser\SwiftSyntaxCDataTypes.h" Checksum="yes" />
```

Comment:
Has this file been there since the beginning of time?  The installer manifests are shared across versions, and we still have active builds on older releases.  Could you please verify that this file exists in 5.3+?

Reply:
The new file existence details as below - 
[5.3](https://github.com/apple/swift/tree/release/5.3/include/swift-c/SyntaxParser) ❌ 
[5.4](https://github.com/apple/swift/tree/release/5.4/include/swift-c/SyntaxParser) ❌ 
[5.5](https://github.com/apple/swift/tree/release/5.5/include/swift-c/SyntaxParser) ❌ 
[5.6](https://github.com/apple/swift/tree/release/5.6/include/swift-c/SyntaxParser) ✔️ 

So this file started appearing from 5.6 onwards.